JPA Extensions concerning the search index (https://jira.pvtool.org/b…

moritz-bayer opened the pull request:
…rowse/VKO-3070)

Nach besten Wissen und Gewissen in dem Projekt umgesetzt.

Commit:
JPA Extensions concerning the search index (https://jira.pvtool.org/browse/VKO-3070)

stefanseifert reviewed (changes requested):
thanks for the PR, lgtm in general.

i've noticed that you've used tabs instead of spaces in all changes to the XML/XSL/XSD files. please convert those two spaces to be in line with the existing code.

Commit:
fixed formatting issue and double search-index entry in platform.xml

fixed code formatting issues found in code review

search-index was written twice/entity due to a double entry in platform.xml. Also fixed.

stefanseifert committed:
add all new fields to XML schema
regenerate schema code with latest SDK xsd.exe
prepare documentation of new fields

stefanseifert commented:
not all new fields were added to the XSD - i've added them and regenerated the schema code with a current version of xsd.exe.

please update the README file as well - i've already added lines for the new fields, please enter a documentation instead of the "???"

stefanseifert committed:
set version to 4.9.0.0
